debian/cortx-motr-dev.install

```diff
@@ -543,18 +543,16 @@ usr/include/motr/xcode/ut/test_gccxml_simple_xc.h
 usr/include/motr/xcode/ut/test_gccxml_xc.h
 usr/include/motr/xcode/xcode.h
 usr/include/motr/xcode/xcode_attr.h
-usr/lib/x86_64-linux-gnu/libm0isc.la
-usr/lib/x86_64-linux-gnu/libmotr-altogether.la
-usr/lib/x86_64-linux-gnu/libmotr-helpers.la
-usr/lib/x86_64-linux-gnu/libmotr-net-test.la
-usr/lib/x86_64-linux-gnu/libmotr-ut.la
-usr/lib/x86_64-linux-gnu/libmotr-xcode-ff2c.la
-usr/lib/x86_64-linux-gnu/libmotr-xcode-ff2c.so.0.0.0
-usr/lib/x86_64-linux-gnu/libmotr-xcode-ff2c.so
-usr/lib/x86_64-linux-gnu/libmotr-xcode-ff2c.so.0
-usr/lib/x86_64-linux-gnu/libmotr.la
-usr/lib/x86_64-linux-gnu/libtestlib.la
-usr/lib/x86_64-linux-gnu/pkgconfig/motr.pc
+usr/lib/*/libm0isc.la
```

Comment:
is there no other way instead of using "*" here ?

Reply:
Not that I'm aware of. Why? As far as I understand, it's quite a standard way.

Reply:
drwxr-xr-x  3 root root   4096 Nov 24 05:16 usrmerge
drwxr-xr-x  2 root root   4096 Dec 14 02:22 valgrind
drwxr-xr-x 38 root root  36864 Jan 12 13:08 x86_64-linux-gnu
drwxr-xr-x  2 root root   4096 Oct 13 20:51 xfsprogs
vagrant@ubuntu21:~$ ls -l /usr/lib/

There are lot of other directories in /usr/lib, how it is ensured that x86_64-linux-gnu is used  ?

Reply:
These files are installed from tmp folder at debian/ where the files are installed by `make install` command, so it's safe because there will be only our files, nothing else.

Reply:
How it selects `x86_64-linux-gnu` directory or arm* directory here ?

Reply:
No idea, `make install` does it by itself somehow. And note: it does not create it in CentOS, so looks like it depends on the distribution. It's probably autoconf/automake thing.

Reply:
ok.